mne/viz/backends/_notebook.py

```diff
@@ -10,58 +10,201 @@
 
 class _Renderer(_PyVistaRenderer):
     def __init__(self, *args, **kwargs):
-        self.tool_bar_state = True
         self.tool_bar = None
-        self.actions = dict()
+        self.dock_width = 300
+        self.dock = None
+        self.actions = None
+        self.widgets = None
         kwargs["notebook"] = True
         super().__init__(*args, **kwargs)
 
+    def _add_widget(self, layout, widget):
+        from ipywidgets import HBox
```

Comment:
Rather than nesting all `ipywidgets` imports, can we avoid importing `mne.viz.backends._notebook` elsewhere in the code until we need to do so (i.e., nest its import to delay trying to import `ipywidgets`)?

Reply:
still to be un-nested